packages/strapi-plugin-graphql/services/Mutation.js

```diff
@@ -195,27 +195,37 @@ module.exports = {
 
       // Resolver can be a function. Be also a native resolver or a controller's action.
       if (_.isFunction(resolver)) {
+        const normalizedName = _.toLower(name);
+        const primaryKey =
+          action &&
```

Comment:
same here :)

Just for me, can you just do a simple if statement for this to avoid ternary plus property acces on the same line :) 
```js
let primaryKey;
if (plugin) {
  primaryKey = ...
} else {
   primaryKey = 
}
```